frontend/www/js/omegaup/arena/contest_practice.ts

```diff
@@ -11,13 +11,20 @@ import arena_ContestPractice, {
 OmegaUp.on('ready', () => {
   time.setSugarLocale();
   const payload = types.payloadParsers.ContestPracticePayload();
+  // The hash is of the form `#problems/${alias}`.
+  const problemMatch = /#problems\/([^/]+)/.exec(window.location.hash);
+  const problemAlias = problemMatch?.[1] ?? null;
+  let problem: ActiveProblem | null = null;
+  if (problemAlias) {
+    problem = { alias: problemAlias, runs: [] };
+  }
   const contestPractice = new Vue({
     el: '#main-container',
     components: { 'omegaup-arena-contest-practice': arena_ContestPractice },
     data: () => ({
       problemInfo: null as types.ProblemInfo | null,
       problems: payload.problems as types.NavbarContestProblem[],
-      problem: null as ActiveProblem | null,
+      problem: problem as ActiveProblem | null,
```

Comment:
```suggestion
      problem,
```

aquí `problem` ya tiene el tipo correcto desde L17.

Reply:
Listo